text/2019-08-21-supported-platforms.md

```diff
@@ -0,0 +1,230 @@
+      + The target can successfully run `cargo build` and `cargo build --release`
+        on TiKV without patches or modifications
+      + The target can produce development builds of TiKV, they do not
+        necessarily run
+* **Tier 3 to Tier 2:**
+      + The target can host and run TiKV as a service, and serve requests
+      + The target can actively particpate in a TiKV cluster containing TiKV
+        nodes running on tier 1 targets
+      + The target can pass TiKV's test suite and benchmark suite
+      + The proposal is submitted by a regular contributor to TiKV, and they
+        state they intend to maintain support
+* **Tier 2 to Tier 1:**
+      + The target has reliably passed TiKVs test and benchmark suites for a
+        period of 3 months
+      + The target can use an interactive debugger on TiKV and documentation is present
+      + The target supports at least one fuzzer and fuzzing has been performed
+      + The target is used as a development machine by at least one maintainer
+        or three regular contributors
+
+### Tier Definitions
+
+This RFC proposes the following definitions for the tiers of platforms and targets:
+
+* **Deployment Platforms**
+      + **Tier 1**:
+        - CentOS 7.x.x on x86-64 on x84-64 with SSE4.2
+        - Ubuntu 16.04 through 19.04 on x84-64 with SSE4.2
+      + **Tier 2**:
+        - Any Linux Kernel >3.x with glibc >2.17 on x86-64 with SSE4.2
+        - Any Linux Kernel >3.x with glibc >2.17 on ARM64/AARCH64
+      + **Tier 3**:
+        - Any Linux Kernel >3.x with glibc >2.17 on x86-64 without SSE4.2
+        - Any Linux Kernel >3.x with musl libc on X86-64 or ARM64/AARCH64, with
+          or without SSE4.2
+        - The latest Mac OS X release on X86-64 (Caution: Apple offers no
+          written support cycle)
+      + **Not supported**:
+        - Windows
+        - FreeBSD
+        - Any Linux using the 'Unbreakable Enterprise Kernel' by Oracle.
+* **Development Platforms**
+      + **Tier 1**:
+        - Any Linux kernel >3.x on X84-64 with glibc 2.17 with SSE4.2 or without
+        - The latest Mac OS X release on X86-64
+      + **Tier 2**:
+        - Any machine with access to a Docker installation which can support
+          Linux containers
+      + **Tier 3**:
+        - Nix (the package manager) environments
+        - Any Linux Kernel >3.x with musl libc on X86-64 or ARM64/AARCH64 with
+          SSE4.2 or without
+      + **Not supported**:
+        - Windows
+        - FreeBSD
+        - Any Linux using the 'Unbreakable Enterprise Kernel' by Oracle.
+
+This list does not attempt to list all posibilities, it lists all major targets
+we are asked about on a regular basis and special cases we are aware of. If
+something is not listed, it should be considered untested and in the "Not
+supported" tier.
+
+### RFC Effects
+
+If accepted, this RFC would:
+
+* **discourage** adding this data to the TiKV documentation.
```

Comment:
Does "documentation" here mean specifically outside of the repo?

Reply:
Outside of this RFC.